fix for compilation issue when using Intel compilers: "cannot open source file intrin.h"
author: Kenneth Hoste (HPC-UGent)
--- Tools/atomicops.h.orig	2018-01-03 11:44:47.796088542 +0100
+++ Tools/atomicops.h	2018-01-03 11:45:05.956221635 +0100
@@ -83,7 +83,7 @@
 
 }    // end namespace moodycamel
 
-#if (defined(AE_VCPP) && (_MSC_VER < 1700 || defined(__cplusplus_cli))) || defined(AE_ICC)
+#if (defined(AE_VCPP) && (_MSC_VER < 1700 || defined(__cplusplus_cli)))
 // VS2010 and ICC13 don't support std::atomic_*_fence, implement our own fences
 
 #include <intrin.h>
